components/typography/styles.js

```diff
@@ -41,6 +41,27 @@ export const bodyStandardStyles = css`
 	}
 `;
 
+export const bodyCompactStylize = (tag) => css`
+	${tag} {
+		font-size: 0.8rem;
+		font-weight: 400;
+		line-height: 1.2rem;
+	}
+	:host([skeleton]) ${tag}.d2l-skeletize::before {
+		bottom: 0.3rem;
+		top: 0.3rem;
+	}
+	:host([skeleton]) ${tag}.d2l-skeletize-paragraph-2 {
+		max-height: 2.4rem;
+	}
+	:host([skeleton]) ${tag}.d2l-skeletize-paragraph-3 {
+		max-height: 3.6rem;
+	}
+	:host([skeleton]) ${tag}.d2l-skeletize-paragraph-5 {
+		max-height: 6rem;
+	}
+`;
+
 export const bodyCompactStyles = css`
```

Comment:
Can this just call:

```css
bodyCompactStylize(css`.d2l-body-compact`)
```

Reply:
Yup! Still need to figure out why Lit syntax highlighting and style linting doesn't like this though.